src/Http/Controllers/MeetupController.php

```diff
@@ -14,7 +14,7 @@ class MeetupController extends Controller
 {
     public function index(): View
     {
-        $meetups = Meetup::latest()->with(["user"])->paginate(20);
+        $meetups = Meetup::query()->latest()->with(["user"])->paginate(20);
```

Comment:
Move maybe number of items per page into some variable placed in config file.

Reply:
Not better a move this option to front .And make it possible for the user to choose pagination 20, 40, 60, e.g.

Reply:
Maybe but starting value should be loaded.